src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:41521/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:41521/project/issues?id=sample&pullRequest=5673&issues=AWcwnvypFotwTOOEO5h9&open=AWcwnvypFotwTOOEO5h9)

[ISSUE_KEY]: AWcwnvypFotwTOOEO5h9